lib/modules/manager/gomod/__fixtures__/3/go-mod

```diff
@@ -133,4 +133,3 @@ replace (
 )
 
 toolchain go1.22.1
```

Comment:
this should be removed, ho would remove it. should never lower than go directive

Reply:
Updated in https://github.com/renovatebot/renovate/pull/32546